plugins/pay.c

```diff
@@ -824,25 +824,32 @@ static struct command_result *add_shadow_route(struct command *cmd,
 	const jsmntok_t *chan, *best = NULL;
 	size_t i;
 	u64 sample = 0;
-	u32 cltv, best_cltv;
+	/* FIXME: Use route_info's cltv_expiry_delta member instead */
```

Comment:
The gossip protocol restricts that to a u16 btw...

Reply:
Didn't notice that subtlety, thanks